fda-model/structures/StructureDamageResult.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace structures
+{
+    public class StructureDamageResult
```

Comment:
in theme with the naming convention of this project, we might name this "DeterministicStructureDamageResult"